config/crd/bases/osp-director.openstack.org_openstackbackups.yaml

```diff
@@ -3983,6 +3983,15 @@ spec:
                                           GB - use RootDisk.DiskSize instead
                                         format: int32
                                         type: integer
+                                      evictionStrategy:
+                                        default: LiveMigrate
+                                        description: VirtualMachineRunStrategy is
```

Comment:
maybe we should override the descriptions?

Reply:
updated